### ISSUE TITLE
Uninitialized when constructing a default b2Vec2?

### ISSUE THREAD

**Issue by TTimo** [pybox2d/pybox2d#66]
Just noticed this:

>>> from Box2D import *
>>> b2Vec2()
b2Vec2(0,-2.52435e-29)

Thought it could be some uninitialized memory problem ..  or something else?

**Comment by kne** [comment]
Thanks for the report, @TTimo, but I'm not seeing that on 2.7 or 3.4 with the current master:

```python
In [1]: import Box2D

In [2]: Box2D.b2Vec2()
Out[2]: b2Vec2(0,0)
```
Can you give me more details about your environment so I can try to reproduce it?

**Comment by TTimo** [comment]
That first report was no my MacOS X system. On Debian sid GNU/Linux x64 I see this:

```
Python 2.7.10+ (default, Oct 10 2015, 09:11:24) 
[GCC 5.2.1 20151028] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from Box2D import *
>>> b2Vec2()
b2Vec2(3.22483e-38,0)
>>> import Box2D
>>> Box2D.__version__
'2.3b0'
```

**Comment by kne** [comment]
Sorry, it's been a while since I've delved into this source. It appears that this is the expected behavior in the underlying Box2D library: the default constructor of b2Vec2 does not initialize x/y.

I think I'll modify the behavior for pybox2d as that's hardly user-friendly (but then again the rest of the library isn't much better, so maybe it's a good expectation-setting experience? :) ).

**Comment by TTimo** [comment]
:)

**Comment by kne** [comment]
Fixed in d3ea69d94c6ff7a80d8415d04b0ed8066c451fb4
